resources/text/units.md

```diff
@@ -0,0 +1,96 @@
+# Units
```

Comment:
I think this discussion is not about font "units" or "coordinates" in general, but more specifically about _variation axis_ coordinates.